src/net_processing.cpp

```diff
@@ -3961,13 +3961,11 @@ void PeerLogicValidation::EvictExtraOutboundPeers(int64_t time_in_seconds)
             AssertLockHeld(cs_main);
 
             // Ignore non-outbound peers, or nodes marked for disconnect already
```

Comment:
Should be along the following "ignore non-outbound-full-relay peers"?
Or probably even better: "Consider only outbound full-relay peers not already marked for disconnection"?

Reply:
Done.